apps/andi/test/integration/andi_web/live/data_dictionary_form_test.exs

```diff
@@ -571,12 +571,21 @@ defmodule AndiWeb.DataDictionaryFormTest do
 
       html = render_click(data_dictionary_view, "add_data_dictionary_field", %{})
 
-      assert [
-               {"Top Level", technical_id},
-               {"one", _},
-               {"one > one-one", new_eligible_parent_id},
-               {"two", _}
-             ] = get_all_select_options(html, ".data-dictionary-add-field-editor__parent-id select")
+      expected_options = [
+        "Top Level",
+        "one > one-one",
+        "one",
+        "two"
+      ]
+
+      select_options = get_all_select_options(html, ".data-dictionary-add-field-editor__parent-id select")
+
+      Enum.each(select_options, fn select_option ->
+        option_name = Tuple.to_list(select_option) |> Enum.at(0)
```

Comment:
It looks like you want one of the items in the `select_option` tuple. This is best done with pattern matching in Elixir:
```elixir
Enum.each(select_options, fn {option_name, _} ->
```